packages/toolpad-app/src/server/toolpadAppBuilder.ts

```diff
@@ -285,6 +285,7 @@ export function createViteConfig({
           'superjson',
           'zod',
         ],
+        exclude: ['@mui_toolpad_runtime', '@mui_toolpad_canvas'],
```

Comment:
Does this only work with
```
@mui_toolpad_runtime
```
or also with
```
@mui/toolpad/runtime
```
?

Reply:
I can try `@mui/toolpad/runtime` in case it works.